cargo-pgx/README.md

```diff
@@ -802,3 +803,30 @@ Postgres, so loading two versions of the shared library will cause trouble.
 These scenarios are:
 - when using shared memory
 - when using query planner hooks
+
+## Compiler Version Dependence
+
+The version of the Rust compiler and toolchain used to build `cargo-pgx` must be
+the same as the version used to build your extension.
+
+Several subcommands (including `cargo pgx schema`, `cargo pgx test`, `cargo pgx
+install`, ...) will produce an error message if these do not match.
+
+Although this may be relaxed in the future, currently schema generation involves
+`dlopen`ing the extension and calling `extern "Rust"` functions on
+`#[repr(Rust)]` types. Generally, the appropriate way to fix this is reinstall
+`cargo-pgx`, using a command like the following
+
+```shell script
+$ cargo install --locked cargo-pgx
```

Comment:
Cargo will only reinstall an already installed binary with the same version if `--force` is passed.
```suggestion
$ cargo install --force --locked cargo-pgx
```

Reply:
Oh, nice catch!